auth-web/src/components/auth/manage-business/AddBusinessDialog.vue

```diff
@@ -191,6 +218,22 @@ export default class AddBusinessDialog extends Vue {
     return CommonUtils.isFirmNumber(this.businessIdentifier)
   }
 
+  get isStaffAccount (): boolean {
+    return this.currentUser.roles.some((role) => (role === Role.Staff || role === Role.GOVMAccountUser))
```

Comment:
Role GOVM account User? I don't think that's right

Reply:
You'll have to look at the org's `type_code`

Reply:
Travis, do you have sample code?

Reply:
```
  protected get isSbcStaffAccount (): boolean {
    return this.currentOrganization?.orgType === Account.SBC_STAFF
  }
  ```
  if we're dealing with Affiliations to SBC_STAFF org.. if we're not that will need to be changed

Reply:
I pass in isSbcStaffAccount from EntityManagement to AddBusinessDialog. This prevents the need to fetch staff account in AddBusinessDialog and passes the unit test files.

Reply:
` return this.currentUser.roles.some((role) => (role === Role.Staff || role === Role.GOVMAccountUser))`

->
` return this.currentUser.roles.some((role) => (role === Role.Staff))`